shared/src/sys/out.rs

```diff
@@ -82,12 +82,17 @@ pub mod vm {
 
 pub mod network {
     use crate::clock::ChainEpoch;
+    use crate::sys::TokenAmount;
 
     #[derive(Debug, Copy, Clone)]
     #[repr(packed, C)]
     pub struct NetworkContext {
         /// The current epoch.
-        pub network_curr_epoch: ChainEpoch,
+        pub epoch: ChainEpoch,
+        /// The current time (seconds since the unix epoch).
```

Comment:
this currently is hardwired no?

Reply:
For now, yes.

Reply:
Ah, no, it's not hardwired. I need to fix that todo.

Reply:
the ts is wires in the fevm lotus branch, nothing to fix.